src/rendering/custom-element-widget.js

```diff
@@ -153,6 +162,17 @@ function makeUpdate() {
   };
 }
 
+function makeDestroy() {
+  return function destroyCustomElement(element) {
+    let proxiedProps = element.cycleCustomElementMetadata.propertiesProxy;
+    for (let prop in proxiedProps) { if (proxiedProps.hasOwnProperty(prop)) {
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).